packages/react/src/feature-hub-context.tsx

```diff
@@ -26,13 +34,20 @@ export interface FeatureHubContextValue {
   addUrlForHydration?(url: string): void;
 }
 
+export type FeatureHubContextValue = FeatureHubContextProviderValue;
```

Comment:
I will add the following deprecating warning in a separate `fix` commit, where I also need to replace the deprecated interface in `packages/demos/src/server-side-rendering/integrator.node.tsx`.

```ts
/**
 * @deprecated Use {@link FeatureHubContextProviderValue} or
 * {@link FeatureHubContextConsumerValue} instead.
 */
```

Otherwise `@feature-hub/demos` would also get a minor version bump, and the feature would be mentioned in its changelog.

Reply:
If we would allow fast-forward merges, I could even add the commit to this PR. 😉

Reply:
Do as discussed.

Reply:
https://github.com/sinnerschrader/feature-hub/pull/411